Documented hypr-audit sources. (#113)
syslog-ng/syslog-ng.github.io: doc/_admin-guide/060_Sources/031_Hypr/README.md

To follow audit trails restart syslog-ng.

### Example: hypr-audit-trail minimal configuration

```config
source s_hypr {
    hypr-audit-trail(
        url('https://<custom domain>.hypr.com')
        bearer-token('<base64 encoded bearer token>')
    );
};
```

### Example: hypr-audit-trail detailed configuration

```config
source s_hypr {
    hypr-audit-trail(
        url('https://<custom domain>.hypr.com')
        bearer-token('<base64 encoded bearer token>')
        page-size(<number of results to return in a single page>)
        initial-hours(<number of hours to search backward on initial fetch>)
        application-skip-list('HYPRDefaultApplication', 'HYPRDefaultWorkstationApplication')
        log-level('INFO')
        flags(<optional flags passed to the source>)
        ignore-persistence(<yes/no>)
    );
};
```
## Hypr App Audit Trail

The `hypr-app-audit-trail()` monitors the audit trail for one specific RP Application ID. This driver requires the `rp-app-id()` parameter in order to operate.

## Acknowledgements

> These chapters are based on the hypr-audit-trail() content, written by Dan Elder.

Reviewer: ```suggestion
These chapters are based on hypr-audit-trail() documentation written by Dan Elder and Axoflow.
```
